packages/redis/src/RedisStreamSink.ts

```diff
@@ -0,0 +1,87 @@
+import {
+    IOutputSink,
+    IPublishedMessage,
+    IOutputSinkGuarantees,
+    IRequireInitialization,
+    IDisposable,
+    IComponentContext,
+    makeLifecycle,
+    Lifecycle,
+    failSpan,
+    DefaultComponentContext,
+    OutputSinkConsistencyLevel,
+    RetrierContext,
+} from "@walmartlabs/cookie-cutter-core";
+import { Span, Tags, Tracer } from "opentracing";
+
+import { redisClient, IRedisClient, RedisMetadata, IRedisOutputStreamOptions } from ".";
+import { RedisOpenTracingTagKeys } from "./RedisClient";
+import { ParserError, AggregateError } from "redis";
+
+export class RedisStreamSink
```

Comment:
can you add a function to creates the RedisStreamSink to the root index file of this module? for all the other packages we follow the convention that there is a function the creates the sink and returns an `IOutputSink` type and then we don't export the concrete types from the package and keep them hidden as impl details